flipperdevices/flipperzero-firmware
# Desktop: update first start for release 0.60.2

# What's new

- Desktop: update first start for release 0.60.2

# Verification 

- Compile and upload
- Check first start screen
- Check collateral damage: subghz not tx screen 

# Checklist (For Reviewer)
